lib/vm-zfs

```diff
@@ -267,6 +267,7 @@ zfs::clone(){
     # generate a short uuid and create snapshot if no custom snap given
     if [ -z "${_snap}" ]; then
         vm::confirm_stopped "${_old}" || exit 1
+        vm::has_saved_state "${_old}" && util::err "guest appears to be suspended, stop it before cloning"
```

Comment:
This I don't understand. If you cannot clone a suspended VM, why support it?

Reply:
To prevent cloning suspended VMs, exit with an error if the VM is suspended. Is something wrong?

Reply:
It somewhat logically contradicts "Prohibit cloning suspended VMs to avoid MAC address conflicts". The message above implies that you can clone when it is stopped. Is the MAC address to sole problem or the suspend state?

Reply:
Updated the commit message.